apps/backend/src/modules/project-groups/libs/helpers/generate-project-group-key.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { changeCase } from "~/libs/helpers/helpers.js";
+
+const generateProjectGroupKey = ({
```

Comment:
We don't need this helper, just concat the string where it's used `${projectId}_${changeCase(name, "snakeCase")}`

Reply:
Okay, I just thought it would be too much to copy this three times

Reply:
Oh, I thought we only needed this in one place. In this case yes, better to have this helper

Reply:
I returned it